tests/lookup/tests.py

```diff
@@ -90,6 +91,24 @@ def test_iterator(self):
             ['Article 4'],
             transform=attrgetter('headline'))
 
+    def test_iterator_invalid_chunk_size(self):
```

Comment:
Maybe this is the best place for the test at this time, but I'm confused why it's here because it doesn't have anything to do with queryset lookups, which is what the test module name "lookup" seems to imply.